README.md

```diff
@@ -16,6 +16,10 @@ Use the following instructions to install Codewind with your choice of editor:
 3. [Eclipse Che plugin](https://github.com/eclipse/codewind-che-plugin)
 
 ## Building Codewind from the source
+1. Clone the `codewind` repository.
+2. Run the `./script/build.sh` script to run the Codewind build, or run the `./run.sh` script to build and start Codewind.
+
+You can also build Codewind from the IDEs:
```

Comment:
You can't build Codewind from the IDEs.  You build Codewind using the build scripts and then you can build one of the IDEs for Codewind using the steps in the IDE's readme.